README.md

```diff
@@ -130,8 +130,8 @@ The following endpoints exist:
 * prod, CDN: https://epr.elastic.co
 * staging, no CDN: https://epr-staging.ea-web.elastic.dev
```

Comment:
I wonder if we shouldn't keep only endpoints with CDN.

Reply:
I think for debugging purpose it is quite nice to have these documented.

Reply:
As discussed off-PR, we remove them from this particular README.

Reply:
Removed in 19c82f59e313373c59b61f358ceeded7f59655fa.